packages/astro/src/runtime/astro.ts

```diff
@@ -1,41 +0,0 @@
-import type { AstroComponent, AstroComponentFactory } from '../internal';
-
-import { spreadAttributes, defineStyleVars, defineScriptVars } from '../internal';
```

Comment:
This file was deleted because this introduced a circular dependency between `src/runtime/astro.ts` and `src/internal/index.ts`. This was just moved into `internal`.

Naming is hard.